tests/tsan.supp

```diff
@@ -1,51 +1,91 @@
 # ThreadSanitizer (tsan) suppression file
 #
-race:libqpid-proton-proactor
-deadlock:libpython
+#race:libqpid-proton-proactor
+#deadlock:libpython
 
-race:qd_vlog_impl
-deadlock:qd_vlog_impl
-race:qd_log_entity
+# proton false positive?
+# TBD: discuss with proton devs
+deadlock:pni_timer_set
```

Comment:
Full disclosure ;P Deadlock detector in TSan appears to be lacking, https://github.com/google/sanitizers/issues/1258

In LLVM Clang, there are two versions of the detector, that can be switched at compile time (of llvm).